src/components/index.js

```diff
@@ -1,3 +1,4 @@
 import Instructions from './Instructions.component'
-
+import Counter from './counter.component'
```

Comment:
typically the naming convention for components has them start with capital letters